### PR TITLE
Removed parent parameter from update request/

### PR DESCRIPTION
**Pull request by martijnvg**
PR for #4538

### CODE REVIEW

File: docs/reference/migration/migrate_2_0.asciidoc [hunk 1]
```diff
@@ -102,4 +102,8 @@ Some query builders have been removed or renamed:
 * `filtered(...)` removed. Use `filteredQuery(...)` instead.
 * `inQuery(...)` removed.
 
+=== Parent parameter on update request
 
+The `parent` parameter has been removed from the update request. Before 2.x it just set the routing parameter. The
+`routing` setting should be used instead. The `parent` setting was confusing, because it has the impression that the parent
+a child documents points to can be changed and this is untrue.
```
Comment (rjernst) [inline docs/reference/migration/migrate_2_0.asciidoc]: "and this is untrue" -> "but this is not true"

File: docs/reference/migration/migrate_2_0.asciidoc [hunk 2]
```diff
@@ -102,4 +102,8 @@ Some query builders have been removed or renamed:
 * `filtered(...)` removed. Use `filteredQuery(...)` instead.
 * `inQuery(...)` removed.
 
+=== Parent parameter on update request
 
+The `parent` parameter has been removed from the update request. Before 2.x it just set the routing parameter. The
+`routing` setting should be used instead. The `parent` setting was confusing, because it has the impression that the parent
```
Comment (rjernst) [inline docs/reference/migration/migrate_2_0.asciidoc]: has -> had

**Comment by rjernst** [comment]
LGTM, just some minor grammar notes.

**Comment by martijnvg** [comment]
Thanks @rjernst, I will apply your grammer notes.

**Comment by clintongormley** [comment]
Reverted by #9612

**Pull request merged**